setup.py

```diff
@@ -64,16 +80,67 @@
     )
 
 
+class BuildExt(build_ext):
+    """ A custom build extension for adding compiler-specific options.
+
+    """
+    c_opts = {
+        'msvc': ['/EHsc']
+    }
+
+    def initialize_options(self):
+        build_ext.initialize_options(self)
+        self.debug = False
+
+    def build_extensions(self):
+        ct = self.compiler.compiler_type
+        opts = self.c_opts.get(ct, [])
+        for ext in self.extensions:
+            ext.extra_compile_args = opts
+        build_ext.build_extensions(self)
+
+
+class Install(install):
+    """ Calls the parser to construct a lex and parse table specific
+        to the system before installation.
+
+    """
+
+    def run(self):
+        try:
+            from enaml.core.parsing import write_tables
+            write_tables()
+        except ImportError:
+            pass
+        install.run(self)
+
+
+class Develop(develop):
+    """ Calls the parser to construct a lex and parse table specific
+        to the system before installation.
+
+    """
+
+    def run(self):
+        try:
+            from enaml.core.parsing import write_tables
+            write_tables()
+        except ImportError:
+            pass
+        develop.run(self)
+
+
 setup(
     name='enaml',
-    version='0.9.8',
+    version=__version__,
     author='The Nucleic Development Team',
     author_email='sccolbert@gmail.com',
     url='https://github.com/nucleic/enaml',
     description='Declarative DSL for building rich user interfaces in Python',
     long_description=open('README.rst').read(),
-    requires=['atom', 'PyQt', 'ply', 'kiwisolver'],
-    install_requires=['distribute', 'atom >= 0.3.8', 'kiwisolver >= 0.1.2', 'ply >= 3.4'],
+    requires=['future', 'atom', 'PyQt', 'ply', 'kiwisolver'],
+    install_requires=['setuptools', 'future', 'atom',
```

Comment:
This requires atom>=0.4 and kiwisolver>=0.1.3. These should be released to pip as currently they must be built using the git repos. Building fails w/o these. 

Also I had to install 'wheel' for 'bdist_wheel' to work (python 3.5.2 on linux x86-64). Not sure if that should be in the requirements.

Reply:
As those versions are not yet officialy released I would prefer to wait on them, but this is true it needs to be updated before enaml next release.

Installing wheel to build wheel distribution is normal and does not need to be added here.